mergesort/mergesort.py

```diff
@@ -1,22 +1,27 @@
 
 __all__ = ["merge_sort"]
 
-def merge_sort(a,b,n,m):
+def merge_sort(a,b): 
     """
-    Creates a new ordonated succesion with all elements from two given ordonated series
+    Creates a new sorted succesion with all elements from two given sorted series
     
     Args:
         a: the first succesion
         b: the second succesion
-        n: the number of elements in a
-        m: the number of elements in b
     Returns:
         c: the created succesion using merge sort
     """
 
     c=[]
     i=0
     j=0
+    n=len(a)
```

Comment:
naming: you're running out of letters for vars ;) a_size = len(a).. b_size = len(b).. "tell a story"